### PR TITLE
gluon-web-mesh-vpn-fastd: clarify how to set the default value in site.conf

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: docs/features/vpn.rst [hunk 1]
```diff
@@ -32,8 +32,9 @@ increase throughput, although in practice the gain is minimal.
 
 **Site configuration:**
 
-1) Install ``gluon-web-mesh-vpn-fastd`` in ``site.mk``
+1) Add the package ``gluon-web-mesh-vpn-fastd`` in ``site.mk``
```
Comment (rotanid) [inline docs/features/vpn.rst]: as we would advise to use "features", please adjust your PR accordingly

File: docs/features/vpn.rst [hunk 2]
```diff
@@ -32,8 +32,9 @@ increase throughput, although in practice the gain is minimal.
...
 2) Set ``mesh_vpn.fastd.configurable = true`` in ``site.conf``
+3) optional add ``null`` to the ``mesh_vpn.fastd.methods`` table if you want "Performance mode" as default (not recommended)
```
Comment (rotanid) [inline docs/features/vpn.rst]: 1) *optionally
2) is performance mode automatically the default, if you add it? your comment suggests this.

Comment (rubo77) [reply docs/features/vpn.rst]: done

File: docs/features/vpn.rst (hunk 1 above)
Comment (rubo77) [reply docs/features/vpn.rst]: done

**Pull request merged**